### ISSUE TITLE
trained succesfully but did not create model file?

### ISSUE THREAD

**Issue by skywo1f** [qqwweee/keras-yolo3#754]
I went through the training process and reached epoch 100:
Epoch 100/100
3757/3757 [==============================] - 170s 45ms/step - loss: 8.9247 - val_loss: 8.8859

I am expecting a file called model_data/trained_weights_final.h5 but no such file exists. Was it saved somewhere else, or did I miss a parameter in my training script?

**Comment by daniele-salerno** [comment]
You should have a folder named logs. Check inside logs/000/
